core/settings.py

```diff
@@ -216,6 +220,11 @@
         environment=ENV or "local",
     )
 
+# add debug toolbar when not running tests
```

Comment:
Can we also tie this to the DEBUG setting, so that it doesn't get included in the installed apps / middleware in other environments?

Reply:
That's the case by default:
>[SHOW_TOOLBAR_CALLBACK](https://django-debug-toolbar.readthedocs.io/en/latest/configuration.html#toolbar-options)
... used for determining whether the toolbar should show or not ... `DEBUG` must be set to True and the IP of the request must be in `INTERNAL_IPS`